apps/demos/menuMeta.json

```
@@ -4098,16 +4098,15 @@
       },
       {
         "Name": "Stepper",
+        "Breadcrumb": true,
         "Badge": "New",
         "Equivalents": "dxStepper, dx-stepper, stepper",
         "Demos": [
           {
             "Title": "Overview",
             "Name": "Overview",
             "Widget": "Stepper",
-            "DemoType": "Web",
-            "Badge": "Roadmap",
-            "RoadmapSurveyUrl": "https://www.devexpress.com/support/surveys/devextreme-roadmap-stepper.xml"
+            "DemoType": "Web"
```

Equivalents field is missed

---

Let's declare other possible field with default values
```suggestion
            "DocUrl": "",
            "MvcDescription": "",
            "NetCoreDescription": "",
            "MvcAdditionalFiles": [],
```